CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+We love pull requests. Here's a quick guide.
+
+### 1. Fork, then clone the repo:
+
+    git clone git@github.com:spotify/apollo.git
+
+### 2. Install dependencies & build:
+
+    mvn package
+
+### 3. Write your feature
+
+Ok, so now you can write some code. Have fun!
+
+### 4. Add tests for your feature. Make the tests pass:
+
+    mvn verify
```

Comment:
Maybe also add a note about the `-Pcoverage` profile to check coverage metrics have not dropped.